app/models/rule.rb

```diff
@@ -10,6 +10,13 @@ class Rule < ApplicationRecord
   has_many :comments, dependent: :destroy
   belongs_to :project
 
+  ##
+  # Override `as_json` to include dependent records (e.g. comments, histories)
+  #
+  def as_json(options = {})
+    super.merge({ comments: comments.as_json, histories: history })
+  end
```

Comment:
Should the default for rule.to_json really include everything? Seems heavy.

Reply:
Right now all of the rule attributes are being used (maybe this will change when we add new attributes?) and histories is the minimal required data from `Rule#histories`.

For comments, we could potentially exclude `rule_id`, `id`, `user_id`, and `updated_at` to reduce the amount of data returned here

Reply:
pushed a change that removes those unnecessary comment fields